a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -96,9 +97,24 @@ def get_create_member_flow(self, topology):
         create_member_flow.add(a10_database_tasks.CountMembersWithIP(
             requires=constants.MEMBER, provides=a10constants.MEMBER_COUNT_IP
         ))
+        create_member_flow.add(a10_database_tasks.GetFlavorData(
+            rebind={a10constants.LB_RESOURCE: constants.LOADBALANCER},
+            provides=constants.FLAVOR))
+        create_member_flow.add(server_tasks.MemberFindNatPool(
```

Comment:
Let's move all the NatPool flow tasks into a subflow to cut down on LOC and make it more manageable

Reply:
done